Source/ViewControllers/BrickViewController+UIViewControllerPreviewing.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  BrickViewController+ UIViewControllerPreviewing.swift
+//  BrickKit
+//
+//  Created by Aaron Sky on 8/2/17.
+//  Copyright © 2017 Wayfair. All rights reserved.
+//
+
+import UIKit
+
+#if os(iOS)
+extension BrickViewController: UIViewControllerPreviewingDelegate {
+    open func previewingContext(_ previewingContext: UIViewControllerPreviewing, viewControllerForLocation location: CGPoint) -> UIViewController? {
+        guard let indexPath = brickCollectionView.indexPathForItem(at: location) else {
+            return nil
+        }
+        let brick = brickCollectionView.brick(at: indexPath)
+        let viewController = brick.previewingDelegate?.previewViewController
+        if let previewing = viewController as? BrickViewControllerPreviewing {
+            previewing.sourceBrick = brick
+        }
+        return viewController
+    }
+    
+    open func previewingContext(_ previewingContext: UIViewControllerPreviewing, commit viewControllerToCommit: UIViewController) {
```

Comment:
This can technically be unit-tested to. Just call this method with a mock `BrickViewControllerPreviewing` and make sure `commit` was called